notebooks/105-language-quantize-bert/105-language-quantize-bert.ipynb

```diff
@@ -196,11 +198,31 @@
    },
    "outputs": [],
    "source": [
-    "ir_model_xml = onnx_model_path.with_suffix(\".xml\")\n",
-    "ir_model_bin = onnx_model_path.with_suffix(\".bin\")\n",
+    "ir_model_xml_32 = onnx_model_path.with_name(onnx_model_path.stem + \"_32\").with_suffix(\".xml\")\n",
```

Comment:
On CPU there is no performance difference between FP16 and FP32. I tried benchmark_app with GPU on my TGL laptop, and there is a nice progression from FP32->FP16->INT8. It is good to show that.
I recommend adding a short explanation here about why we're converting to both FP16 and FP32 and mention in the benchmark_app section that FP16 is only faster on GPU. 
The model loads pretty quickly on GPU so IMO it is useful to add a line for the benchmark_app commands with (untested old API syntax) `device = "GPU" if "GPU" in ie.available_devices else "CPU"` so it will automatically show the best performance on computers with a supported iGPU.